2.10/CHANGELOG-v2.10.rst

```diff
@@ -8,6 +8,405 @@ This changelog describes changes since Ansible 2.9.0.
   :local:
   :depth: 2
 
+v2.10.6
+=======
+
+.. contents::
+  :local:
+  :depth: 2
+
+Release Summary
+---------------
+
+Release Date: 2021-01-26
+
+`Porting Guide <https://docs.ansible.com/ansible/devel/porting_guides.html>`_
+
+Added Collections
+-----------------
+
+- community.fortios (version 1.0.0)
+
+Ansible-base
+------------
+
+Ansible 2.10.6 contains Ansible-base version 2.10.1.
```

Comment:
Need to fix

Reply:
Fixed in https://github.com/ansible-community/ansible-build-data/pull/55/commits/6cb2fb8c45616fdc777ecac6ec9926d6cec3947c and https://github.com/ansible-community/ansible-build-data/pull/55/commits/b06fde3873f35a15e19e013927be8bca5e8d7c41